wagtailnhsukfrontend/templates/wagtailnhsukfrontend/panels.html

```diff
@@ -0,0 +1,6 @@
+<div class="nhsuk-panel{{ class }}">
+  {% if value.title %} <!-- it has a header -->
+  <h3 {{ titleclass }}>{{ value.title }}</h3>
```

Comment:
If you did this:
```python
<h3 class="{% if value.type == "labeled"}nhsuk-panel-with-label__label{% endif %}">{{ value.title }}</h3>
```
it would remove the need for titleclass logic in the get_context function